understanding/20/no-keyboard-trap.html

```diff
@@ -1,127 +1,147 @@
-               the applet. Instructions describing the keystroke used to exit the applet are provided
-               prior to the applet as well as within the applet itself.</dd>
-         <dt>A modal dialog box</dt>
-         <dd>A web application brings up a dialog box. At the bottom of the dialog are two buttons,
-               Cancel and OK. When the dialog has been opened, focus is trapped within the dialog;
-               tabbing from the last control in the dialog takes focus to the first control in the
-               dialog. The dialog is dismissed by activating the Cancel button or the OK button.</dd>
-      </dl>
-      
-   </section>
-   
-   <section id="resources">
-      <h2>Resources for No Keyboard Trap</h2>
-      
-      
-   </section>
-   
-   <section id="techniques">
-      <h2>Techniques for No Keyboard Trap</h2>
-      
-      
-      <section id="sufficient">
-         <h3>Sufficient Techniques for No Keyboard Trap</h3>
-         
-         
-         <ul>
-            
-            <li>
-               									         
-               <a href="../Techniques/general/G21" class="general">Ensuring that users are not trapped in content</a>
-               								       
-            </li>
-            
-         </ul>
-         
-      </section>
-      
-      <section id="advisory">
-         <h3>Additional Techniques (Advisory) for No Keyboard Trap</h3>
-         
-         
-      </section>
-      
-      <section id="failure">
-         <h3>Failures for No Keyboard Trap</h3>
-         
-         
-         <ul>
-            
-            <li>
-               									         
-               <a href="../Techniques/failures/F10" class="failure">Failure of Success Criterion 2.1.2 and Conformance Requirement 5 due to combining
-                  multiple content formats in a way that traps users inside one format type
-               </a>
-               								       
-            </li>
-            
-         </ul>
-         
-      </section>
-      
-   </section>
-   
-</body>
-</html>
+
+	<head>
+		<meta charset="UTF-8">
+		</meta>
+		<title>Understanding No Keyboard Trap</title>
+		<link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove" />
+	</head>
+
+	<body>
+		<h1>Understanding No Keyboard Trap</h1>
+
+		<section id="brief">
+			<h2>In brief</h2>
+			<dl>
+				<dt>Goal</dt>
+				<dd>Keyboard users don't get stuck.</dd>
+				<dt>What to do</dt>
+				<dd>Ensure users always know how to navigate away from components.</dd>
+				<dt>Why it's important</dt>
+				<dd>People who rely on the keyboard often have no other means to navigate.</dd>
+			</dl>
+
+		</section>
+
+		<section id="intent">
+			<h2>Intent of No Keyboard Trap</h2>
+
+
+			<p>The intent of this success criterion is to ensure that that content does not “trap” the
+				keyboard focus within subsections of content on a web page or app. It is common to restrict
```

Comment:
the "It is common..." insert here seems out of place. I'd move it to the next paragraph, and change this back to talking about the *problem*